.travis.yml

```diff
@@ -1,18 +1,23 @@
 ---
 dist: trusty
 language: node_js
-branches:
-    only:
-        - master
-matrix:
-    fast_finish: true
-    include:
-        - os: linux
-          node_js: 8
-          env: CXX=g++-4.8
-          # @TODO macOS build is much slower
-        - os: osx
-          node_js: 8
+node_js: 8
+addons:
+  apt:
+      sources:
+          - ubuntu-toolchain-r-test
+      packages:
+          - gcc-4.8
+          - g++-4.8
```

Comment:
Maybe we can use a more recent gcc (https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/test?field.series_filter=trusty) to speed thing a lit bit (to be tested)